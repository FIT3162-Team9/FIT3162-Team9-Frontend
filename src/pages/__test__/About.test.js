import React from 'react';
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import About from '../About';

it("About page renders correctly", () => {
    const tree = shallow(<About />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  