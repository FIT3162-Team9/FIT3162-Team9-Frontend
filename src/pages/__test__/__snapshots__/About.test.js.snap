// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page renders correctly 1`] = `
<WithStyles(ForwardRef(Container))
  className="makeStyles-container-1"
  maxWidth="lg"
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    spacing={3}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      lg={12}
      md={12}
      xs={12}
    >
      <About />
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Container))>
`;
